documentation/staging/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -86,16 +113,16 @@ The usage of the create script is as follows:
 ```shell
 $ sh create-domain.sh -h
 ```
-```
-usage: create-domain.sh -o dir -i file -u username -p password [-k] [-e] [-h]
+```text
+usage: create-domain.sh -o dir -i file -u username -p password [-m wdt|wlst] [-n encryption-key] [-e] [-v] [-h]
   -i Parameter inputs file, must be specified.
   -o Ouput directory for the generated properties and YAML files, must be specified.
   -u User name used in building the image for WebLogic domain in image.
   -p Password used in building the image for WebLogic domain in image.
+  -m WebLogic configuration mode. Either 'wdt' or 'wlst', optional. Defaults to 'wdt'.
```

Comment:
I suspect it'd be a little less error prone if the configuration were mode an 'input', or if the wdt and wlst source code locations were moved to the command line.  We use the input file to set the model & wlst location, but use the command line to choose between model & wlst...

Reply:
make sense. Thanks. I will move the mode to the input file.